src/server/report/result.js

```diff
@@ -113,40 +114,43 @@ function renderResult(resultData, options) {
 		`;
 	}
 
-	const renderPart = (label, partInfo, noChanges, overlay) => {
-		const img = noChanges ? html`
-			<div class="result-no-changes padding">
-				${ICON_TADA}
-				<p>Hooray! No changes here.</p>
-			</div>
-		` : html`<div class="result-diff-container"><img src="../${partInfo.path}" loading="lazy" alt="">${overlay}</div>`;
+	const renderPart = (label, partInfo, overlay) => {
 		return html`
 			<div class="result-part">
 				<div class="result-part-info">
 					<div class="result-part-info-spacer"></div>
 					<div class="result-part-info-name">${label}</div>
 					<div class="result-part-info-size">(${partInfo.width} x ${partInfo.height})</div>
 				</div>
-				${img}
+				<div class="result-diff-container"><img src="../${partInfo.path}" loading="lazy" alt="">${overlay}</div>
 			</div>
 		`;
 	};
 
-	const overlay = (options.showOverlay && !resultData.passed) ?
+	const goldenExists = (resultData.info.golden !== undefined);
+
+	const overlay = (goldenExists && options.showOverlay && !resultData.passed) ?
```

Comment:
Need to hide the overlay if there's no golden... since there won't be a diff to show.